src/modules/psql_connector.py

```diff
@@ -1,15 +1,20 @@
-from psycopg2 import connect, InterfaceError
+from asyncpg import connect, InterfaceError, create_pool
 import logging
+import asyncio
+from async_timeout import timeout
 
 
 class PSQLConnector():
     def __init__(self, conf):
-        self.conn = self._connect(conf)
+        loop = asyncio.get_event_loop()
+        self.conn = loop.run_until_complete(self._connect(conf))
         self.sync_type = conf["sync_type"]
+        self.conf = conf
+        self.pool = loop.run_until_complete(self._create_pool())
 
     @staticmethod
-    def _connect(conf):
-        conn = connect(
+    async def _connect(conf):
```

Comment:
If we don't need this anymore, pls consider removing it

Reply:
I already done that in new commit